File: python/001-salary-increase.py
salary = float(input("Insira o valor do salario: "))
percentage = float(input("Insira a porcentagem do aumento: "))

increase = (salary * percentage) / 100
new_salary = salary + increase

print("Valor do aumento: R$ {:.2f}".format(increase))
print("Valor do novo salario: R$ {:.2f}".format(new_salary))

File: c/001-salary-increase.c
#include <stdio.h>

int main() {
    float salary;
    float percentage;

    printf("Insira o valor do salário: ");
    scanf("%f", &salary);
    printf("Insira a porcentagem de aumento: ");
    scanf("%f", &percentage);

    float increase = (salary * percentage) / 100;
    float new_salary = salary + increase;

    printf("Valor do aumento: R$ %.2f\n", increase);
    printf("Valor do novo salário: R$ %.2f\n", new_salary);

    return 0;
}
